File: abnormalityAccept/src/main/java/com/abnormality/abnormalityaccept/entity/Notice.java
package com.abnormality.abnormalityaccept.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * @author shanh
 * @version 1.0
 * {@code @description:}
 * @since 2025-07-13
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Notice {
    private Long id;
    private String theme;
    private String content;
    private Date time;
    private Integer state;

}

File: vueAbonormalityAccept/src/api/notice.ts
import request from "@/utils/requests.ts";

export interface Notice {
    id: number
    theme: string
    content: string
    time: Date
    state: number
}

// 获取所有公告
export function findAllNotice(){
    return request.get('notice/list',{
        params:{
            pageNum:1,
            pageSize:10,
        }
    })
}